### PR TITLE
Fix build for trusty - locales-all not available

### PR DESCRIPTION
**Pull request by unman**
One of fixes for QubesOS/qubes-issues#2608

### CODE REVIEW

**Comment by unman** [comment]
@marmarek Should I submit separate PRs against release3.2, as well as master, or will you handle this?

**Comment by marmarek** [comment]
`master` only is ok, I'll handle migrating it to `release3.2`.

**Comment by marmarek** [comment]
This is wrong place - `$(shell ...)` will be executed every time `Makefile.builder` gets parsed. You should place it in `source-debian-quilt-copy-in` target, and without `$(shell ...)`.

File: Makefile.builder [hunk 1]
```diff
@@ -10,6 +10,10 @@ ifeq ($(PACKAGE_SET),vm)
   ARCH_BUILD_DIRS := archlinux
 endif
 
+ifeq ($(DIST),trusty)
+  $(shell sed -i /locales-all,/d  $(CHROOT_DIR)/$(DIST_SRC)/debian/control)
+endif
+
```
Comment (marmarek) [inline Makefile.builder]: This is wrong place - `$(shell ...)` will be executed every time `Makefile.builder` gets parsed. You should place it in `source-debian-quilt-copy-in` target, and without `$(shell ...)`.

**Pull request merged**